### DISCUSSION TITLE
Which energy in LAMMPS should be compared to VASP DFT results?

### DISCUSSION THREAD

**Discussion by PolyuWeldingSpock** [deepmodeling/deepmd-kit#2792] [Q&A]
Hi everyone,

I trained a DP model based on VASP AIMD results. Then I used this DP potential to run LAMMPS on my system. I randomly selected some snapshots from LAMMPS trajectories and do the self-consistent calculations in VASP. I wanted to compare the energy averaged on each atom. For VASP results, I chose energy(sigma->0) value in OUTCAR:
```shell
  energy  without entropy=    -1054.40748036  energy(sigma->0) =    -1054.40748036
```  
But I do not know which compared LAMMPS energy value should be chosen, is it should be TotEng or just PotEng?
```shell
   Step         PotEng         KinEng         TotEng          Temp          Press          Volume    
         0  -1057.5992      8.5699476     -1049.0292      300            9054.8718      3339.2557    
         1  -1057.4598      8.4350513     -1049.0248      295.27781      9420.1881      3339.2557    
```
Hope someone could help. Thanks in advance.

**Comment by owen-rett** [comment]
Potential energy would be the comparison point here; VASP's "energy without entropy" term does not include kinetic energy contributions, so comparing to lammp's total energy (which does), wouldn't make sense.

**Reply by PolyuWeldingSpock** [reply]
Thanks for your reply. @owen-rett 

So which energy in VASP SCF calculation can be compared to TotEng in LAMMPS? I noticed that there are three energies in VASP OUTCAR.
```shell
  FREE ENERGIE OF THE ION-ELECTRON SYSTEM (eV)
  ---------------------------------------------------
  free  energy   TOTEN  =     -1054.40748036 eV

  energy  without entropy=    -1054.40748036  energy(sigma->0) =    -1054.40748036
```
Should be free energy TOTEN?

**Comment by njzjz** [comment]
"Energy without entropy" is inconsistent with forces. In MD simulations, the potential energy is expected to be always consistent with forces. Thus, nothing can be compared with "energy without entropy". See also https://github.com/deepmodeling/dpdata/discussions/411.

**Reply by PolyuWeldingSpock** [reply]
Thanks for your reply! @njzjz 

It seems like that if I want to compare the LAMMPS `TotEng` with VASP. I should choose ` free  energy   TOTEN  =     -1054.40748036 eV` in VASP OUTCAR.
